src/components/TheFondue.vue

```diff
@@ -11,7 +16,11 @@
 			@getFont="$emit('getFont', $event)"
 			@getExampleFont="$emit('getExampleFont', $event)"
 		/>
-		<a href="https://pixelambacht.nl" target="_blank" class="logo"
+		<a
+			href="https://pixelambacht.nl"
+			target="_blank"
+			class="logo"
+			aria-label="PixelAmbacht"
```

Comment:
This could be an [sr-only](https://webaim.org/techniques/css/invisiblecontent/) span.

Reply:
I don't think so, it's the link that leads to my site. I'd like to keep that around for everyone ;-)